pages/test_analytics/ci_environments.md.erb

```diff
@@ -0,0 +1,55 @@
+# CI Environments
+
+Buildkite Test Analytics collectors automatically detect common continuous integration (CI) environments.
```

Comment:
This whole thing sort of implies we ought to have a _collectors_ page that describes the concept in general, then lists all of the collectors: RSpec, Jest, `collector.sh`, and more that come along. It'd also be a good place to link to third-party collectors, when those eventually happen.

```suggestion
Buildkite Test Analytics [collectors](/docs/test-analytics/collectors) automatically detect common continuous integration (CI) environments.
```

Reply:
Makes sense. Not sure how much there is to say about them though..

Reply:
Ah right. There's no collectors homepage, is there? In one of Oly's mockups, we do have subheadings in the left nav, and could easily make a page that has like one sentence -- 'Use collectors to tend test information to Test Analytics' or some such, re-use what I think is a reusable template Rails thingo that Oly made to list collectors on the TA homepage.

There's no a lot to say about them, but this ^^ might help with UX traveling around this part of our docs. 🤔

Reply:
Yeah. I don't feel strongly about it, but would suggest merging without for now.